package-lock.json

```
@@ -11547,8 +11547,8 @@
       }
     },
     "wet-boew": {
-      "version": "github:wet-boew/wet-boew#34615150762ba0c038fa584aa0ca8ee013d95e66",
-      "from": "github:wet-boew/wet-boew#v4.0.53",
+      "version": "github:wet-boew/wet-boew#169efffa90b8b577e3d0f651280d9a60a3558c7e",
```

This hash would need to be updated 

```suggestion
      "version": "github:wet-boew/wet-boew#cf657677b9dbaba961991c21bcdc2242dbf9256f",
```

---

Fixed. Thanks